sentinel.go

```diff
@@ -205,6 +205,20 @@ func (sc *Sentinel) Addrs() (string, []string) {
 	return sc.primAddr, secAddrs
 }
 
+// SentinelAddrs returns the currently known network address of sentinels
```

Comment:
Wording it this way sounds a little better :)

```
SentinelAddrs returns the addresses of all known sentinels.
```

Reply:
Sure